### PR TITLE
[update] Multi-node authentication

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

Included the details to add mask length for trust auth.
Slack thread: https://iobeam.slack.com/archives/C03URQS7NVD/p1671642653422369

# Links

Fixes https://github.com/timescale/docs/issues/1910

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [x] Is the content free from typos?
- [x] Does the content use plain English?
- [x] Does the content contain clear sections for concepts, tasks, and references?
- [x] Have any images been uploaded to the correct location, and are resolvable?
- [x] If the page index was updated, are redirects required
      and have they been implemented?
- [x] Have you checked the built version of this content?

### CODE REVIEW

File: timescaledb/how-to-guides/multinode-timescaledb/multinode-auth.md [hunk 1]
```diff
@@ -274,18 +332,24 @@ When you have created the certificates and keys, and moved all the files into th
...
+
     ```txt
     ssl = on
     ssl_ca_file = 'root.crt'
     ssl_cert_file = 'server.crt'
     ssl_key_file = 'server.key'
     ```
-    **OPTIONAL:** If you want the access node to use certificate authentication for login, make these changes on the access node as well.
+
+    **OPTIONAL:** If you want the access node to use certificate authentication
```
Comment (Loquacity) [inline timescaledb/how-to-guides/multinode-timescaledb/multinode-auth.md:346]:
```suggestion
1. <optional />If you want the access node to use certificate authentication
```

Maybe? Not sure.

**Review by Loquacity — APPROVED** [review]

**Review by Loquacity — APPROVED** [review]

File: timescaledb/how-to-guides/multinode-timescaledb/multinode-auth.md [hunk 2]
```diff
@@ -46,38 +49,58 @@ this method in production. It is not a secure method of operation.
...
 1.  Connect to the access node with `psql`, and locate the `pg_hba.conf` file:
+
     ```sql
     SHOW hba_file;
     ```
+
 1.  Open the `pg_hba.conf` file in your preferred text editor, and add this
-    line. In this example, the access node is located at IP `192.0.2.20`:
+    line. In this example, the access node is located at IP `192.0.2.20` with a
+    masklength of `32`. You can add one of these two lines:
```
Comment (Loquacity) [inline timescaledb/how-to-guides/multinode-timescaledb/multinode-auth.md:61]: mask length?

File: timescaledb/how-to-guides/multinode-timescaledb/multinode-auth.md [hunk 3]
```diff
@@ -274,18 +332,24 @@ When you have created the certificates and keys, and moved all the files into th
...
+
     ```txt
     ssl = on
     ssl_ca_file = 'root.crt'
     ssl_cert_file = 'server.crt'
     ssl_key_file = 'server.key'
     ```
-    **OPTIONAL:** If you want the access node to use certificate authentication for login, make these changes on the access node as well.
+
+1. <optional />If you want the access node to use certificate authentication
```
Comment (Loquacity) [inline timescaledb/how-to-guides/multinode-timescaledb/multinode-auth.md:346]: This line looks like it needs another whitespace char.

Comment (Rajakavitha1) [reply timescaledb/how-to-guides/multinode-timescaledb/multinode-auth.md]: done

File: timescaledb/how-to-guides/multinode-timescaledb/multinode-auth.md (hunk 2 above)
Comment (Rajakavitha1) [reply timescaledb/how-to-guides/multinode-timescaledb/multinode-auth.md]: updated it to mask length...

**Review by Loquacity — APPROVED** [review]

**Pull request merged**